addScript.js

```diff
@@ -3,8 +3,15 @@
 	Author Tobias Koppers @sokra
 */
 module.exports = function(src) {
-	if (typeof execScript !== "undefined")
-		execScript(src);
-	else
-		eval.call(null, src);
+	try {
+		if (typeof eval !== "undefined") {
+			eval.call(null, src);
+		} else if (typeof execScript !== "undefined") {
+			execScript(src);
+		} else {
+			// TODO error: no eval function available
```

Comment:
As I mention in the description of my pull request, I don't know how these errors should be handled. We could just log them ( `console.log("no eval function available")` ) or throw them (`throw new Error("no eval function available"), so that they have to be handled outside of the script loader.